SoftLayer/CLI/virt/os_available.py

```diff
@@ -0,0 +1,28 @@
+"""Get the all Os available."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@environment.pass_env
+def cli(env):
+    """Get the all Os available"""
+
+    table = formatting.KeyValueTable(['Id', 'KeyName', 'Description', 'Price', 'Setup Fee'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+    manager = SoftLayer.OrderingManager(env.client)
+    _filter = {"items": {"prices": {"categories": {"categoryCode": {"operation": "os"}}}}}
+    operations = manager.get_items(storage_filter=_filter, package_id=46)
```

Comment:
Use the `PUBLIC_CLOUD_SERVER` package, as I think that is more inline with what is in the portal. The results are likely very similar, but I think this is the default package now.

```suggestion
    # PUBLIC_CLOUD_SERVER = 835
    operations = manager.get_items(storage_filter=_filter, package_id=835)
```

It might also be a good idea to add this package as a constant in the ordering manager. I'm OK with having the Id here as its not likely to change ever, but having a constant for it would be more "proper"  I think.